python/lsst/meas/base/plugins.py

```diff
@@ -137,23 +137,39 @@ def __init__(self, config, name, schema, metadata):
         SingleFramePlugin.__init__(self, config, name, schema, metadata)
         self.keyProbability = schema.addField(name + "_value", type="D",
                                               doc="Set to 1 for extended sources, 0 for point sources.")
+        self.keyFlag = schema.addField(name + "_flag", type="Flag",
+                                              doc="Set to 1 for any fatal failure.")
```

Comment:
This and all the following minor formatting problems were fixed.  I won't add a comment for each one.